website/promotion/emails.py

```diff
@@ -1,16 +1,30 @@
 """The emails defined by the promotion request package."""
+import logging
+
 from django.conf import settings
 from django.utils.translation import gettext_lazy as _
-
 from utils.snippets import send_email
+from django.template.loader import get_template
+from promotion.models import PromotionRequest
+
+logger = logging.getLogger(__name__)
+
+def send_weekly_overview():
```

Comment:
To send this every week, a management command calling this function should be created, and a cronjob should be added to https://github.com/svthalia/concrexit/blob/c9762db146168569870d2936179e0a92041bb3a4/infra/nixos/concrexit.nix#L230-L250